client/src/app/actions/index.js

```diff
@@ -8,10 +8,14 @@ export const ADD_TO_WISHLIST = 'ADD_TO_WISHLIST';
 export const ADD_ITEMS_TO_FILTERED = 'ADD_ITEMS_TO_FILTERED';
```

Comment:
for the future
follow branch naming strategy
`feature/UIO-48`

Reply:
ok